### PR TITLE
Jane camera button

### PR DESCRIPTION
**Pull request by Jane2353**
Added a button and openCamera function to take photos. Unit tests also made to ensure the camera works correctly.

- Bugfix
- Feature
- Code style update
- Refactor
- Documentation update
- Other (please describe):

## Checklist

- [x] Code follows the code guidelines
- [x] Tests have been added/updated (if applicable)

### CODE REVIEW

**Review by MaksimHamko — DISMISSED** [review]
Everything looks good, maybe a bit too much comments ;)

File: components/Camera/Camera.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as ImagePicker from 'expo-image-picker';
+
+export const openCamera = async () => {
```
Comment (MertzAndreas) [inline components/Camera/Camera.tsx:3]: Question does this function open the camera only? It seems to me that it opens the camera and allows for a picture to be taken. If that's the case I would prefer a different name.

**Review by MertzAndreas — CHANGES_REQUESTED** [review]
Code looks good, I agree with Maksim that the amount of comments are bit overwhelming. 🕺

**Review by MertzAndreas — APPROVED** [review]
Smukt ;)

**Review by Tbjerregaard — APPROVED** [review]

**Pull request merged**